restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -304,6 +304,26 @@ private Query getInsertQueryForTable(ImmutableKeyspaceAndTable info) {
         .join();
   }
 
+  private Query getInsertQueryForTableUsingTtl(ImmutableKeyspaceAndTable info, int ttl) {
```

Comment:
Can we have `int` -> `Integer` and set TTL if the object is not null.. Then we don't have to duplicate the logic in the above method and have simply:

```
private Query getInsertQueryForTable(ImmutableKeyspaceAndTable info) {
    return getInsertQueryForTable(info, null);
}
```